python/tests/refactored_tests/test_decimal_agg0.py

```diff
@@ -0,0 +1,159 @@
+from tests.refactored_tests.test_base_class import Base
+import unittest
+from decimal import Decimal
+
+class DecimalAggregates0(Base):
```

Comment:
instead of a class with many tests here, can you try with two classes and 1 test each?